x/cellarfees/keeper/query_server.go

```diff
@@ -27,6 +27,6 @@ func (k Keeper) ModuleAccounts(c context.Context, req *types.QueryModuleAccounts
 	}
 
 	return &types.QueryModuleAccountsResponse{
-		FeesAddress: k.GetFeesAccount(sdk.UnwrapSDKContext(c)).String(),
+		FeesAddress: k.GetFeesAccount(sdk.UnwrapSDKContext(c)).GetAddress().String(),
```

Comment:
The existing version of this just printed out the entire `ModuleAccount` struct, which would include the address. Is there a reason we want to drop the other info, other than general cleanliness?

Reply:
I think that makes sense. I made this change because of the field name not matching the actual value.